church-services/chuch-services.css

```diff
@@ -0,0 +1,44 @@
+header {
+  background-image: url("../assets/pastor_preaching.jpeg");
+}
+
+.section-title {
+  text-align: center;  
+}
+
+.sermon-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+
+  & .show-more{
+    text-align: center;
+    width: 100%;
+    & button {
+      background-color: transparent;
+      color: var(--accent-color);
+      text-decoration: underline;
+      font-size: var(--fs-lg);
+    }
+  }
+}
+
+
+
+.sermon-card {
```

Comment:
can you move this code to the cards css to make it reusable to all pages? rename the class to something more generic